nemo/collections/llm/bert/model/google_bert.py

```diff
@@ -0,0 +1,367 @@
+import math
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Annotated, Callable, Optional
+
+import torch
+from torch import nn
+
+from nemo.collections.common.tokenizers import AutoTokenizer
+from nemo.collections.llm.bert.model.base import BertConfig, BertModel
+from nemo.collections.llm.utils import Config
+from nemo.lightning import OptimizerModule, io, teardown
+
+
+@dataclass
+class GoogleBertConfig(BertConfig):
```

Comment:
Just curious about the naming, are there different Bert Base or Large than Google's?

Reply:
I don't think so, but I also not sure what could be a better naming ? Any suggestion here ?

Reply:
Just BertBase and BertLarge?

Reply:
sure I'll rename those